### PR TITLE
add gedd.ski site to showcase

### PR DESCRIPTION
**Pull request by geddski**
ahhh yeah!

screenshot-short

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -2293,3 +2293,22 @@
...
+    - JavaScript
+    - Grid
+    - Flexbox
+    - PWA
+    - Productivity
+    - UI
+    - UX
+  built_by: Dave Geddes
+  built_by_url: https://twitter.com/geddski
+  # you know you want to...
```
Comment (DSchau) [inline docs/sites.yml]: 😆

Comment (geddski) [reply docs/sites.yml]: 😁

Comment (geddski) [reply docs/sites.yml]: yep totally gonna optimize it with those image plugins. Gatsby is so dope.

Comment (pieh) [reply docs/sites.yml]: Heh, this is fun, but could you remove that comment? https://next.gatsbyjs.org/docs/site-showcase-submissions/#note-on-featured-sites :)

Comment (geddski) [reply docs/sites.yml]: done :D

**Comment by KyleAMathews** [comment]
Woot!

**Pull request merged**